src/test/scala/com/mongodb/spark/rdd/partitioner/MongoShardedPartitionerSpec.scala

```diff
@@ -16,19 +16,18 @@
 
 package com.mongodb.spark.rdd.partitioner
 
-import org.scalatest.FlatSpec
-
-import org.bson.{BsonDocument, BsonMaxKey, BsonMinKey, Document}
-import com.mongodb.ServerAddress
 import com.mongodb.spark.RequiresMongoDB
+import org.bson.{BsonDocument, BsonMaxKey, BsonMinKey, Document}
 
-class MongoShardedPartitionerSpec extends FlatSpec with RequiresMongoDB {
+class MongoShardedPartitionerSpec extends RequiresMongoDB {
 
   "MongoShardedPartitioner" should "partition the database as expected" in {
     if (!isSharded) cancel("Not a Sharded MongoDB")
     loadSampleDataIntoShardedCollection(5) // scalastyle:ignore
 
-    MongoShardedPartitioner.partitions(mongoConnector, readConfig).length should be >= 2
+    val partitions = MongoShardedPartitioner.partitions(mongoConnector, readConfig)
```

Comment:
Can this test assert more about the Array[MongoPartition] than it's currently doing?